composer/src/main/kotlin/com/gojuno/composer/TestRun.kt

```diff
@@ -181,7 +181,21 @@ private fun pullTestFiles(adbDevice: AdbDevice, test: InstrumentationTest, outpu
             )
         }
 
-private fun saveLogcat(adbDevice: AdbDevice, logsDir: File): Observable<Pair<String, String>> = Observable
+class LogLineParser {
```

Comment:
You don't need class :) 

As you can see, Composer is class-less other than data-classes, just drop a private top-level function :)

Reply:
I can't test it if it's private.

Reply:
good point